feat: section about Enums, file naming convention (#29)
juntossomosmais/frontend-guideline: README.md

┣ ┣ ┃ ┃ ┣ 📜 UserProfile.stories.mdx \
```

#### Scoped Files 

We need to add inside `pages/**/{utils, helpers, context, hooks, etc...}` and use `camelCase` as **Naming Convention**.

> If we need to used this files again in another context or page it need to be moved to `src/{utils, helpers, context, hooks}`.

Reviewer: ```suggestion
If we need to use these files again in another context or page it need to be moved to `src/{utils, helpers, context, hooks}`.
```

Author: thanks!! fix in [91eaf3b](https://github.com/juntossomosmais/frontend-guideline/pull/29/commits/91eaf3b5bc124ec841b837a8113ac43c04012258)

For example:

```sh
┣ 📂 pages \
┣ ┣ 📂 Home \
┣ ┃ ┣ 📂 utils \
┣ ┃ ┣ ┣ 📜 someUtils.js
┣ ┃ ┣ 📂 helpers \
┣ ┣ ┣ ┣ 📜 someHelper.js
┣ ┃ ┣ 📂 hooks \
┣ ┣ ┣ ┣ 📜 useSomeHook.js
```

> 

Reviewer: Suggestion: This example is for the first scenario, maybe we could put it right after the description of it. Markdown got messed up, but I'll try to show what I'm thinking:

```
We need to add inside `pages/**/{utils, helpers, context, hooks, etc...}` and use `camelCase` as **Naming Convention**.

[your exemple here]

If we need to used this files again in another context or page it need to be moved to `src/{utils, helpers, context, hooks}`.
```

Author: Nice suggestion!! I created the examples for when moving these guys to a global context :)
thanks, done in [f642d5f](https://github.com/juntossomosmais/frontend-guideline/pull/29/commits/f642d5f942236ce61c1602bd8b1f76ecaa25d364)

┣ ┣ ┃ ┃ ┣ 📜 UserProfile.stories.mdx \
```

#### Scoped Files 

We need to add inside `pages/**/{utils, helpers, context, hooks, etc...}` and use `camelCase` as **Naming Convention**.

For example:

```sh
┣ 📂 pages \
┣ ┣ 📂 Home \
┣ ┃ ┣ 📂 utils \
┣ ┃ ┣ ┣ 📜 someUtils.js
┣ ┃ ┣ 📂 helpers \
┣ ┣ ┣ ┣ 📜 someHelper.js
┣ ┃ ┣ 📂 hooks \
┣ ┣ ┣ ┣ 📜 useSomeHook.js
```

> If we need use these this files again in another context or page it need to be moved to `src/{utils, helpers, context, hooks}`.

Reviewer: ```suggestion
If we need use these files again in another context or page it need to be moved to `src/{utils, helpers, context, hooks}`.
```

"this" is still there. 😆 
[_suspense song intensifies_]

Author: :hehe:

Author: done :sai-pra-la-bug:

```sh
┣ 📂 src
┣ ┣ 📂 utils \
┣ ┃ ┣ 📜 someUtils.js
┣ ┣ 📂 helpers \
┣ ┃ ┣ 📜 someHelper.js
┣ ┣ 📂 hooks \
┣ ┃ ┣ 📜 useSomeHook.js

> ```

Reviewer: Praise: Nice!

)
```

### 7.7 Avoid compare directly strings

> When know all possible values we can use enum to achieve better readability, control.

Reviewer: ```suggestion
When know all possible values we can use enum to achieve better readability and control.
```

Author: thanks!! fix in [91eaf3b](https://github.com/juntossomosmais/frontend-guideline/pull/29/commits/91eaf3b5bc124ec841b837a8113ac43c04012258)
